fix: check if the config file exists (#1555)
Redocly/redocly-cli: .changeset/hot-vans-smile.md

---
"@redocly/openapi-core": patch
---

> Fix the failure when the path to a not existing config file is provided.

Reviewer: ```suggestion
Improved the experience when the config file doesn't exist or isn't found.
```
